components/Layout.js

```diff
@@ -30,17 +30,24 @@ export default function Layout({
 
       <Container>
         <Header />
-        <main>
+        <Main>
           {children}
-        </main>
+        </Main>
         <Footer />
       </Container>
     </>
   )
 }
 
-const Container = styled('footer', {
+const Container = styled('div', {
   fontFamily: '$sans',
   color: '$forest',
   backgroundColor: '$sand',
-});
\ No newline at end of file
+  flexDirection: 'column',
+  display: 'flex',
+  minHeight: '100%',
+});
+
+const Main = styled('main', {
+  flexGrow: 1
+});
```

Comment:
These styles make it so that when the page content is less than the height of the window, the Footer sticks to the bottom instead of hang up high